tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/js/export-deployment-artifacts/docker-config-dialog.js

```diff
@@ -45,6 +46,28 @@ define(['require', 'lodash', 'jquery'],
                     self.dockerDetailsForm.hide();
                 }
             });
+
+            self.container.find("#docker-push-checkbox").change(function() {
+                var pushDocker = self.container.find("#docker-push-checkbox").is(":checked");
```

Comment:
Get change event and check if it is true/false

Reply:
Fixed in https://github.com/siddhi-io/distribution/pull/752/commits/90bb44445c41a71bd18f5efe9b3338d5358fc112